mta/entities/projects.py

```diff
@@ -306,7 +306,10 @@ class Project(BaseEntity, Updateable):
     def exists(self):
         """Check project exist or not"""
         view = navigate_to(self.parent, "All")
-        return view.projects.exists(self.name)
+        for row in view.table:
```

Comment:
You can do if 

             if self.get_project(name):
                  return True
             return False

Reply:
@sshveta it is not possible. `get_project` is a method of collection not entity.
```
In [1]: col = app.collections.projects

In [2]: y = col.all()[1]

In [3]: y.exists
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-82864ce5a938> in <module>
----> 1 y.exists

~/migration_toolkit_for_applications/windup_integration_test/mta/entities/projects.py in exists(self)
    311         #         return True
    312         # return False
--> 313         if self.get_project(self.name):
    314             return True
    315         return False

AttributeError: 'Project' object has no attribute 'get_project'
```